app/assets/javascripts/staff/events.js

```diff
@@ -7,5 +7,9 @@ $(document).ready(function () {
   $('.cancel-status-change').click(function() {
     $('.status-dropdown').hide();
     $('.btn-nav').show();
-  })
-});
\ No newline at end of file
+  });
+
+  $('.modal').on('keypress', function (e) {
+    if (e.which == 13) $(this).submit();
+  });
```

Comment:
I'm not clear on what the class on modal buttons does. Can you explain that a bit more? Like, which buttons and what exactly is connecting that css class to what behavior?